polar_route/route_calc.py

```diff
@@ -272,23 +272,29 @@ def order_track(df, track_points):
     user_track = pd.DataFrame({'Point': path_point, 'CellID': cell_ids})
     return user_track
 
-def route_calc(route_file, mesh_file):
+def route_calc(df, from_wp, to_wp, mesh):
     """
         Function to calculate the fuel/time cost of a user defined route in a given mesh
 
         Args:
-            route_file (str): Path to user defined route
-            mesh_file (str): Path to mesh with vehicle information
+            df (DataFrame): Route info in dataframe format
+            from_wp (str): Name of start waypoint
+            to_wp (str): Name of end waypoint
+            mesh (json): A Mesh with encoded vehicle information
```

Comment:
```suggestion
            mesh (dict): A Mesh with encoded vehicle information
```

Fantastic, thanks for this!